src/manager.cpp

```diff
@@ -143,9 +143,8 @@ bool Manager::parseOpdsDom(const pugi::xml_document& doc, const std::string& url
     m_totalBooks = strtoull(libraryNode.child("totalResults").child_value(), 0, 0);
     m_startIndex = strtoull(libraryNode.child("startIndex").child_value(), 0, 0);
     m_itemsPerPage = strtoull(libraryNode.child("itemsPerPage").child_value(), 0, 0);
-    m_hasSearchResult = true;
```

Comment:
`m_hasSearchResul` is a member of `Manager` and is public, so part of the API.
Even if, indeed, it is not used in any of our project, we cannot simply not set it.

The `totalResults`/`startIndex`/`itemsPerPage` nodes are present when parsing a OPDS which is a search result. If parsing a root opds flux, they are not present.
So we